05/app.js

```diff
@@ -0,0 +1,37 @@
+function Student (name, surname){
+    this.firstName = name;
+    this.lastName = surname;
+    this.grades = {}
+}
+
+Student.prototype.addGrade = function (subject, grade){
+    if(typeof this.grades[subject] === 'undefined') {
+        this.grades[subject] = [];
+    }
+   this.grades[subject].push(grade);
+}
+Student.prototype.getAverageGrade = function(subject){
+    let sum = 0;
+    let counter = 0;
+    if(typeof subject === 'undefined'){
+        for(const key in this.grades){
+            this.grades[key].forEach(element => sum = sum + element);
+            counter = counter + this.grades[key].length;
+        }
+        console.log(sum/counter);
```

Comment:
Zamiast `console.log()`, powinno być `return`, go sama nazwa funkcji na to wskazuje (jest `get...` zamiast `show...`)